androidClient/app/src/main/java/com/csatimes/dojma/utilities/DHC.java

```diff
@@ -88,6 +83,16 @@
      */
     public static final String UPDATE_SERVICE_HERALD_PAGES = "HERALD_PAGES";
 
+    /**
+     * Firebase node name for moreapps.
+     */
+    public static final String FIREBASE_DATABASE_REFERENCE_MOREAPPS = "moreapps";
```

Comment:
keep all firebase node names in separate class `FirebaseKeys` similar to `SpKeys`. (For now you can add only those you've created)